README.md

```diff
@@ -91,6 +92,32 @@ bwrap --ro-bind /usr /usr \
    /bin/sh
 ```
 
+Sandboxing
+----------
+
+The goal of bubblewrap is to run an application in a sandbox, where it has
+access to less resources.
+
+bubblewrap always creates a new filesystem namespace, and the user can specify
+exactly what parts of the filesystem should be visible in the sandbox.
+Any such directories you specify mounted nodev by default, and can be made readonly.
+
+Additionally you can use these kernel features:
+
+User namespaces: This hides all but the current uid and gid from the
+sandbox. You can also change what the value of uid/gid should be in the sandbox.
+
+IPC namespaces: The sandbox will get its own copy of all the
+different forms of IPCs, like SysV shared memory and semaphores.
+
+PID namespaces: The sandbox will not see any processes outside the sandbox. Additionally, bubblewrap will run a trivial pid1 inside your container to handle the requirements of reaping children in the sandbox.
```

Comment:
...This avoids what is known now as the [Docker pid 1 problem](https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/).